units/host_allocator.go

```diff
@@ -254,33 +255,35 @@ func (j *hostAllocatorJob) Run(ctx context.Context) {
 	const lowRatioThresh = float32(.25)
```

Comment:
I want to make this tunable in future as this is probably somewhat suboptimal, but I've already spent enough time on this and it should be good enough.